data/en/0_star_wars.json

```diff
@@ -16,10 +16,19 @@
         "blocks": [{
                 "type": "Introduction",
                 "id": "introduction",
-                "information_to_provide": [
-                    "Yoda's siblings",
-                    "Total Yearly cost of Rebel Alliance"
-                ]
+                "body": [{
+                    "id": "information-to-provide",
+                    "type": "Basic",
+                    "content": [{
+                        "title": "You will be asked to provide information for the business, including:",
+                        "list": [
+                            "Yoda's siblings",
+                            "Total Yearly cost of Rebel Alliance"
+                        ]
+                    }, {
+                        "description": "<p class=\"venus\">If actual figures are not available, please provide informed estimates.</p>"
```

Comment:
We shouldn't be putting classes in the schema, a `strong` tag would be semantically more appropriate too.